docs/secure-connections/ipsec-vpn.md

```diff
@@ -105,12 +105,12 @@ To connect to Sauce Labs real and virtual devices, assign your Sauce IPSec Proxy
 
 #### Appium and Selenium
```

Comment:
@mdsauce Since this section covers both Appium and Selenium, don't we need to mention both `tunnelName` and `tunnelIdentifier`? Maybe split this into two sections?

cc: @sweeneyskirt-sl @waggledans